vsc_pdf_template_transformer/lib/models/tpl_document.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/widgets.dart';
+
+part 'tpl_document.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplDocument {
+  TplDocument({
+    this.compress = true,
+    this.verbose = false,
+    this.title,
+    this.author,
+    this.creator,
+    this.subject,
+    this.keywords,
+    this.producer,
+  });
+
+  final bool compress;
```

Comment:
All of these fields support expressions - just like any other widget.  We should also have a list of children here, even though Document only supports addPage().